public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer les Dynasties Eyrie",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie sur la clairière du bord de la carte qui a 2+ clairières entre elle et les patries ennemies.<br/><i>2.</i> Mettez 6 <strong>guerriers< /strong> et 1 <strong>perchoir</strong> dans votre patrie.<br/><i>3.</i> Placez n'importe quelle <strong>carte de chef</strong> sur votre emplacement de carte de chef. Gardez l'autre 3 chefs face visible à proximité.<br/><i>4.</i> Placez vos 2 <strong>Vizirs loyaux</strong> sous les emplacements de la colonne Décret comme indiqué sur votre chef actuel.<br/><i> 5.</i> Remplissez vos espaces Perchoirs avec des <strong>perchoirs</strong>, sauf le plus à gauche."
+    },
+    "keepers": {
+      "name": "Gardiens en fer",
+      "summaryTitle": "Joueur les Gardiens",
+      "summary": "<p>En tant que Gardiens de Fer, vous marquez des points en récupérant des <strong>reliques</strong> perdues lors de conflits passés. Vous devrez <strong>extraire</strong> des reliques des forêts, les déplacer vers un <strong>waystation</strong> du même type, puis <strong>récupérez</strong> ces reliques. Que ces reliques vous appartiennent ou appartiennent à la forêt, cependant, est une autre question.</p><p>Comme < strong>Chevaliers dévots</strong> d'un ordre exilé, vous ignorez le premier coup que vous subissez au combat si vous avez à la fois un guerrier et une relique, en attaque ou en défense. Vous pouvez également déplacer des reliques avec vos guerriers.</ p><p>Vos reliques sont des <strong>trophées prisés</strong>, alors gardez-les en sécurité. Chaque fois qu'un ennemi enlève une relique de quelque manière que ce soit, il marque deux points au lieu d'un et le remet dans n'importe quelle forêt .</p><p>Au fil du temps, vous développerez votre <strong>suite</strong>, trois colonnes de cartes qui vous permettent d'effectuer des actions. Cependant, fouiller et récupérer des reliques mettra votre suite en danger. besoin de planifier à l'avance et de faire preuve de prudence prendre des risques pour réussir.</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.